apollos-church-api/config.postgres.yml

```diff
@@ -138,7 +138,7 @@ TABS:
         title: 'All messages'
         relatedNode:
           __typename: ContentChannel
-          id: 8aa1c855-82dc-44a1-bf22-8dd47aff9156
+          apollosId: ContentChannel8aa1c855-82dc-44a1-bf22-8dd47aff9156
```

Comment:
is this what DB ids look like? no delimiter between type and guid?

Reply:
@redreceipt for content_item_category (Content Channel in Rock) yes. We may have just forgot to put in the ":" in the shovel.